README.md

```diff
@@ -1,75 +1,128 @@
-		f.	Run Load().devices_in_use_hourly() to generate the number of devices in use for each hour
-		g.	Run Load().device_load_hourly() to get the load of each device
-		h.	Run Load().total_load_hourly() to get the total community load, segregated into “Domestic”, “Commercial” and “Public” demand types
-	7.	Set up the energy system
-		a.	In your location folder, open the “Simulation” folder
-		b.	Complete the “Energy system inputs.csv” template with the details of your location
-		c.	In your location folder, open the “Scenario” folder
-		d.	Complete the “Scenario inputs.csv” template with the details of your location
-	8.	Perform a simulation
-		a.	Run Energy_System().simulation(start_year, end_year, PV_size, storage_size) with your chosen system
-		b.	Record the outputs as a variable to investigate the outputs in more detail
-		c.	Save the outputs using Energy_System().save_simulation(simulation_name,filename)
-		d.	Open a saved simulation using Energy_System().open_simulation(filename)
-	9.	Input financial information
-		a. 	In your location folder, open the "Impact" folder
-		b. 	Complete the "Financial inputs.csv" with details of your location
-	10.	Input GHG information
-		a. 	In your location folder, open the "Impact" folder
-		b. 	Complete the "GHG inputs.csv" with details of your location
-	11.	Set up the optimisation process
-		a.	In your location folder, open the "Optimisation" folder
-		b.	Complete the “Optimisation inputs.csv” template with the details of your location
-	12.	Perform an optimisation
-		a. 	Run Optimisation().multiple_optimisation_step()
-		b.	Record the outputs as a variable to investigate the outputs in more detail
-		c.	Save the outputs using Optimisation().save_optimisation(optimisation_name,filename)
-		d.	Open a saved optimisation using Optimisation().open_optimisation(filename)
+
+CLOVER minigrid simulation and optimisation for supporting rural electrification in developing countries.
+
+## CLOVER Quick Start Guide
+
+This guide provides a very brief introduction to get your CLOVER installation up and running as quickly as possible following the initial download. The file structure has two main branches: 
+* a python branch, `scripts`, which contains CLOVER's source code which is used to perform simulations and generate outputs,
+* and a data branch, `locations`, which contains informaiton describing the locations being modelled and contains parameters to outline the simulations and optimisations that should be run.
+
+An example location, "Bahraich," in India, is included in the initial download for reference.
+
+### Setting up your Python environment
+
+CLOVER is a scientific package and, as such, uses Python packages that may not have come installed by default on your system. These packages can be easily installed, provided that you are connected to the internet, either using `pip`, the python package manager, or `conda`, a virtual-environment-based system:
```

Comment:
Should we recommend one approach over the other? I use conda and am under the impression it is friendlier for more casual users

Reply:
Don't mind :stuck_out_tongue: . On my chromebook, which is basically Linux, I use pip, as it's what came with the computer, but you have to use conda in certain places like the HPC. So maybe conda..?

Reply:
Someone can work out how to use pip if they're really keen :stuck_out_tongue: